src/main/java/uk/gov/companieshouse/web/accounts/controller/accountselector/SelectAccountTypeController.java

```diff
@@ -36,7 +36,6 @@
     public String getTypeOfAccounts(@PathVariable String companyNumber, Model model) {
 
         model.addAttribute("typeOfAccounts", new TypeOfAccounts());
-        addBackPageAttributeToModel(model, companyNumber);
```

Comment:
This same change will need to be made on the post method, too. Otherwise, if binding result errors are present on submission, a back link will appear on screen when the page re-renders.